### DISCUSSION TITLE
Is social-app smatphone (iOS or android) edition useable for federation/after migtating PDS?

### DISCUSSION THREAD

**Discussion by itaru2622** [bluesky-social/atproto#2413] [General Discussion]
anybody tried social-app smatphone (iOS or android) edition to use for federation / self hosting PDS env? I checked official app UI but no component found to configure PDS endpoint which web edition has at the first page(signin/signup).

so I'm wonderIng if it is useable in the above cases.

**Comment by devinivy** [comment]
You should be able to use your self-hosted PDS on the iOS and Android apps.  Here's what it looks like to use your own PDS from the sign-in screen:

https://github.com/bluesky-social/atproto/assets/3150233/8a446026-574c-42aa-8146-1bd69fad581c

**Reply by itaru2622** [reply]
@devinivy thank you for responce and movie. I fogot check signin page but looking for UI after signed-in.
